lib/spack/spack/main.py

```diff
@@ -606,14 +606,9 @@ def __call__(self, *argv, **kwargs):
 
         out = StringIO()
         try:
-            if sys.platform == 'win32':
-                with winlog(out):
-                    self.returncode = _invoke_command(
-                        self.command, self.parser, args, unknown)
-            else:
-                with log_output(out):
-                    self.returncode = _invoke_command(
-                        self.command, self.parser, args, unknown)
+            with log_output(out):
+                self.returncode = _invoke_command(
+                    self.command, self.parser, args, unknown)
```

Comment:
(question) all the changes in this file are just cleanup - right? As in, not essential to fix the problem.